contracts/cw1-subkeys/src/contract.rs

```diff
@@ -30,6 +30,7 @@ use cw_storage_plus::Bound;
 const CONTRACT_NAME: &str = "crates.io:cw1-subkeys";
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+#[entry_point]
```

Comment:
Can you use `#[cfg_attr(not(feature = "library"), entry_point)]` as you did for whitelist?

Reply:
Sure. I did that only when currently necessary. I agree that it's more robust / forward-looking to do it by default.